oauth2_provider/migrations/0011_refreshtoken_token_family.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.2 on 2024-08-09 16:40
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('oauth2_provider', '0010_application_allowed_origins'),
```

Comment:
```suggestion
        ('oauth2_provider', '0010_application_allowed_origins'),
        migrations.swappable_dependency(settings.REFRESH_TOKEN_MODEL)
```
I think this needs to get added. Same question I have outstanding with @dopry on another pending PR.

Because of swappable models I believe the automatically generated migrations have to have this dependency manually added.

Reply:
`0001_inital.py` is using `oauth2_settings.REFRESH_TOKEN_MODEL` for a relation, so `from oauth2_provider.settings import oauth2_settings` and then using `oauth2_settings.REFRESH_TOKEN_MODEL` here?

Reply:
Let's leave it as is for now and then think about a separate PR to understand and update all the migrations to deal with swappable models appropriately. The main issue I've seen is when someone has been using DOT for a while already and then tries to switch to a swappable model after having already populated a "standard" model. When I've done it, I've tended to start over from scratch with an empty database and manually deal with adding applications, etc.